lib/decidim/participatory_documents/version.rb

```diff
@@ -2,8 +2,8 @@
 
 module Decidim
   module ParticipatoryDocuments
-    VERSION = "0.2.2"
-    DECIDIM_VERSION = "0.27.5"
-    COMPAT_DECIDIM_VERSION = [">= 0.27.0", "< 0.28"].freeze
+    VERSION = "0.3.0"
+    DECIDIM_VERSION = "0.28"
```

Comment:
```suggestion
    DECIDIM_VERSION = "0.28.1"
```
Let's upgrade to 0.28.1